### PR TITLE
Fix spelling mistake on word 'weekly'

### PR DESCRIPTION
**Pull request by peteryates**

### CODE REVIEW

File: safeguarding-and-remote-teaching-during-the-coronavirus.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ In some areas, schools may also be able to seek support from their local authori
 
 ### Providing pastoral care remotely
 
-Helping parents, carers and pupils to make a weakly plan or structure that includes time for education, playing and relaxing is important to reduce stress and anxiety for families.
```
Comment (lukechaput) [inline safeguarding-and-remote-teaching-during-the-coronavirus.md:102]: monthly

File: safeguarding-and-remote-teaching-during-the-coronavirus.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ In some areas, schools may also be able to seek support from their local authori
...
+Helping parents, carers and pupils to make a weekly plan or structure that includes time for education, playing and relaxing is important to reduce stress and anxiety for families.
```
Comment (jodyarup) [inline safeguarding-and-remote-teaching-during-the-coronavirus.md:102]: change from 'weekly' to 'frequently'

Comment (rjlynch) [inline safeguarding-and-remote-teaching-during-the-coronavirus.md:102]: Thought's on changing `weekly` to `frequent`

Comment (catherinegilhooly) [inline safeguarding-and-remote-teaching-during-the-coronavirus.md:102]: Should say monthly instead of weekly

Comment (peteryates) [reply safeguarding-and-remote-teaching-during-the-coronavirus.md]: I've made this change

File: safeguarding-and-remote-teaching-during-the-coronavirus.md (hunk 1 above)
Comment (peteryates) [reply safeguarding-and-remote-teaching-during-the-coronavirus.md:102]: Done

**Pull request merged**